sdk/storage/azure-storage-queue/azure/storage/queue/_queue_client.py

```diff
@@ -120,6 +120,19 @@ def __init__(
         self._client._config.version = get_api_version(api_version)  # type: ignore [assignment]
         self._configure_encryption(kwargs)
 
+    def __enter__(self):
```

Comment:
Queue doesn't have `pyi` files so I guess we need to document the full type for enter/exit here. Reminder this is the full proper type hint for exit:
https://github.com/python/typeshed/blob/567b488fc28978642bfeb70b29f11453b0ff124c/stdlib/imp.pyi#L48